examples/00-basic-examples/EDB_Create_VIA.py

```diff
@@ -17,9 +17,12 @@
 from pyaedt.generic.general_methods import generate_unique_name
 start = time.time()
 if os.name == "posix":
-    aedb_path = os.path.join("/tmp", generate_unique_name("Edb_custom") + ".aedb")
+    tmpfold = os.environ["TMPDIR"]
```

Comment:
Note that on Linux `$TMPDIR` is not guaranteed to exist.  Suggest using `get` and using `/tmp` as backup since it's the [Filesystem Hierarchy Standard](http://refspecs.linuxfoundation.org/fhs.shtml).

```suggestion
    tmpfold = os.environ.get("TMPDIR", '/tmp')
```

Also, (nitpick here), tmpfold sounds like temporary folder, which is Windows GUI related and Linux users will think it's strange.  Directory is more cross platform (see [SO discussion](https://stackoverflow.com/questions/5078676)).